modules/nf-core/multiqc/main.nf

```diff
@@ -1,53 +0,0 @@
-process MULTIQC {
```

Comment:
we are using multiqc for a lot of standardised reporting across all pipelines, such as tool version, run commands etc. . Is there any reason why you can't leave it in?

Reply:
I wasn't aware of this when I started developing oncoanalyser a couple of years ago so it had be removed. Plans for to include after first release! See here for some further context: https://github.com/nf-core/oncoanalyser/pull/29#discussion_r1680605946

Reply:
Created an issue to complete this work following the initial release:

- https://github.com/nf-core/oncoanalyser/issues/81